scripts/python/tao_build.py

```diff
@@ -389,6 +391,11 @@ def _write(line, cmd="build"):
                 _action_env("BUILD_WITH_MKLDNN", "1")
             if args.aarch64:
                 _action_env("BUILD_WITH_AARCH64", "1")
+        else:
+            if args.platform_alibaba and args.blade_gemm:
```

Comment:
Nit: Are these configurations related to `args.platform_alibaba` capable to be moved to the internal repo?

Reply:
Currently the two version of build.py  are both with these configurations